CHANGELOG.md

```diff
@@ -13,7 +13,9 @@
 
 ### Enhancements
 
-* Lorem ipsum.
+* Utilities gain `Any` from object store and base64 encoding from sync.
+* Initial support for query serialisation.
+  PR [#2947](https://github.com/realm/realm-core/pull/2947).
```

Comment:
It'd be worth listing the new query features supported by the parser here as well. That'll make it easier for the JS binding to mention them in its release notes when it picks up this change.